datapreview.tex

```diff
@@ -8,13 +8,13 @@ \section{Data Previews and Data Release 1}
 \item Data Preview 2 (DP2): Based on a full reprocessing of all science-grade LSSTCam data taken during commissioning.
 \item Data Release 1 (DR1): Based on the first six months of LSST data. 
 \end{itemize}
-Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), the Data Previews will necessarily be limited in their area and temporal coverage relative to a full Data Release. 
+Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:transition}), the Data Previews will necessarily be limited in their area and temporal coverage relative to a full Data Release. 
 
 The data products that comprise the Data Previews and Data Releases are produced by the LSST Science Pipelines \citep{2019ASPC..523..521B,2018PASJ...70S...5B}.
 For an introduction to the LSST data products, see \citet{RubinDataProductsAbridged} and for a detailed description, see the LSST \dpdd{},  \citedsp{LSE-163}.
 Each Data Preview and LSST Data Release will be accompanied by its own release-specific DPDD\footnote{For an example data release DPDD, see the online DP0.2 documentation {\url{https://dp0-2.lsst.io/data-products-dp0-2/}}.}, giving e.g. the  database schema for the catalogs included in that dataset.
 Table~\ref{tab:data-preview-summary} provides a summary of the expected early science data products available in DP0, DP1, DP2 and the LSST Data Release 1.
-All LSST data products will be subject to the embargo periods described in  \citeds{DMTN-199}.
+All LSST data products will be subject to the embargo periods described in  \citeds{DMTN-199}; 30 days during commissioning and 80 hours during operations for pixel data.
```

Comment:
We are working on an amendment to DMTN-199 that would define the 80hr embargo for all survey mode observations (ie including SV), but it's not finished yet. We shoudl be able to bring RTN-011 into alignment with that in the _next_ update.

Reply:
Great - we can issue a minor release of V6.0 (this release) of RTN-011 as soon as you have that